maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/common/Constants.java

```diff
@@ -5,4 +5,8 @@ private Constants() {}
     public static final String WRN_MSG_REASSESSMENT = "A reassessment is required";
     public static final String WRN_MSG_INCOMPLETE_ASSESSMENT = "Incomplete assessment has been cleared due to this change.";
 
+    public static final String APPLICANT = "APPLICANT";
```

Comment:
@karlbaker02 @Mhowell494 Is it worth reusing the ApplicantType enum from the Evidence Service instead of declaring string constants here? This means we would need to move the ApplicantType Enum from Evidence service to commons jar.